src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -48,6 +48,43 @@ protected function tearDown()
         }
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The provided cwd does not exist. Command is currently ran against getcwd(). This behaviour is deprecated since version 3.4 and will be removed in 4.0.
+     */
+    public function testInvalidCwd()
+    {
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
```

Comment:
should be `if ('\\' === DIRECTORY_SEPARATOR) {` to test for Windows, as everywhere else in the code base.

Reply:
Done